libs/brain/input-otp/src/lib/brn-input-otp.component.ts

```diff
@@ -0,0 +1,127 @@
+import { BooleanInput, NumberInput } from '@angular/cdk/coercion';
+import {
+	booleanAttribute,
+	Component,
+	computed,
+	forwardRef,
+	input,
+	model,
+	numberAttribute,
+	signal,
+} from '@angular/core';
+import { ControlValueAccessor, FormsModule, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { ChangeFn, TouchFn } from '@spartan-ng/brain/forms';
+import { provideBrnInputOtp } from './brn-input-otp.token';
+
+export const BRN_INPUT_OTP_VALUE_ACCESSOR = {
+	provide: NG_VALUE_ACCESSOR,
+	useExisting: forwardRef(() => BrnInputOtpComponent),
+	multi: true,
+};
+
+export type InputMode = 'text' | 'tel' | 'url' | 'email' | 'numeric' | 'decimal' | 'search';
```

Comment:
In case we need to support more (seeing form TODO comment on Line 61-62):
```suggestion
export type InputMode = 'text' | 'tel' | 'url' | 'email' | 'numeric' | 'decimal' | 'search' | (string & Record<never, never>);
```
This TypeScript trick would let users pass arbitrary strings in _addition_ to these strings.

Reply:
I forgot to remove the TODO when i only had `numeric`. The types should be complete (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode), i only left out `none`. Thus are the same types supported by https://github.com/guilhermerodz/input-otp?tab=readme-ov-file#props